rest/server_context.go

```diff
@@ -41,6 +41,9 @@ const kStatsReportInterval = time.Hour
 const kDefaultSlowQueryWarningThreshold = 500 // ms
 const KDefaultNumShards = 16
 
+// fromConnStrWarningThreshold determines the amount of time it should take before we warn about parsing a connstr (mostly for DNS SRV resolution)
```

Comment:
I
```suggestion
// fromConnStrWarningThreshold determines the amount of time it should take before we warn about parsing a connstr (mostly for DNS resolution)
```
I think there's other DNS resolution and the call happens to check if SRV records are there but it can do other things.